src/cmds/play.c

```diff
@@ -103,21 +120,39 @@ int main(int argc, char **argv) {
 
 	callback = &fd_callback;
 
-	while (-1 != (opt = getopt(argc, argv, "nsh"))) {
+	while (-1 != (opt = getopt(argc, argv, "nshf:"))) {
 		switch (opt) {
 		case 'h':
 			print_usage();
 			return 0;
 		case 's':
 			callback = &sin_callback;
 			break;
+		case 'f':
+#if !PLAY_FROM_FLASH
```

Comment:
I think this option is ok even for big platforms. For example, this can be used in debug purpose to play audio from RAM.